expandvars.py

```diff
@@ -102,7 +102,14 @@ def getenv(var, indirect, environ, var_symbol="$"):
     return val
 
 
-def expand(vars_, nounset=False, environ=os.environ, var_symbol="$"):
+def expand(
+    vars_,
+    nounset=False,
+    environ=os.environ,
+    var_symbol="$",
+    var_symbol_require_suffix=False,
+    disable_escape=False,
```

Comment:
Suggesting a better API:

```suggestion
    expand_surrounded_vars_only=False,
    escape_char="\\",  # To disable escaping, pass None
```